src/notation/view/percussionpanel/percussionpanelmodel.cpp

```diff
@@ -301,16 +299,21 @@ void PercussionPanelModel::setUpConnections()
         }
     });
 
-    if (!audioSettings()) {
-        return;
+    if (audioSettings()) {
+        audioSettings()->trackInputParamsChanged().onReceive(this, [this](InstrumentTrackId trackId) {
+            if (trackId != currentTrackId()) {
+                return;
+            }
+            updateSoundTitle(trackId);
+        });
     }
 
-    audioSettings()->trackInputParamsChanged().onReceive(this, [this](InstrumentTrackId trackId) {
-        if (trackId != currentTrackId()) {
-            return;
-        }
-        updateSoundTitle(trackId);
-    });
+    if (configuration()) {
```

Comment:
Actually this null check should be redundant (and otherwise it should be in `useNotationPreview()` too)